vault/resource_pki_secret_backend_intermediate_cert_request.go

```diff
@@ -268,7 +274,7 @@ func pkiSecretBackendIntermediateCertRequestCreate(ctx context.Context, d *schem
 
 	// Fields only used when we are generating a key
 	if !(intermediateType == keyTypeKMS || intermediateType == consts.FieldExisting) {
-		intermediateCertAPIFields = append(intermediateCertAPIFields, consts.FieldKeyType, consts.FieldKeyBits)
+		intermediateCertAPIFields = append(intermediateCertAPIFields, consts.FieldKeyType, consts.FieldKeyBits, consts.FieldSignatureBits)
```

Comment:
Signature bits field is used when generating the certificate so I don't believe it belongs here.

Reply:
Fixed.